src/game_inventory.cpp

```diff
@@ -359,7 +368,12 @@ class comestible_inventory_preset : public inventory_selector_preset
             }, _( "QUENCH" ) );
 
             append_cell( [ p, this ]( const item_location & loc ) {
-                return good_bad_none( p.fun_for( get_comestible_item( loc ) ).first );
+                const item it = get_comestible_item( loc );
```

Comment:
This line makes a full copy of the item (expensive). Should be a reference instead.